torch/_inductor/ir.py

```diff
@@ -2573,7 +2592,94 @@ def is_channels_last_stride_ordered(self):
         order = [len(order)] + order
         return self.is_stride_ordered(order)
 
+    @staticmethod
+    def _pad_strides(in_strides, size, align=DEFAULT_ALIGN):
+        """
+        The padding does not change stride order but makes sure all strides larger
+        than the threshold are multiple of align.
+        """
+        if len(in_strides) == 0:
+            return in_strides
+
+        # a loose check for channels last
+        if not config.pad_channels_last and len(in_strides) == 4 and in_strides[1] == 1:
```

Comment:
I think there is a helper elsewhere in this file for this check, reuse or refactor it?